test/benchee/formatters/csv_integration_test.exs

```diff
@@ -35,19 +37,25 @@ defmodule Benchee.Formatters.CSVIntegrationTest do
           "Sleep longer" => fn -> :timer.sleep(20) end
         }, configuration
 
-
         assert File.exists?(filename)
+        assert File.exists?(@raw_benchmark)
 
-        [header, _row_1, _row_2] = filename
-                                 |> File.stream!
-                                 |> CSV.decode
-                                 |> Enum.take(3)
-
-        [first_header | _rest] = header
-        assert first_header == "Name"
+        assert_header(filename)
+        assert_header(@raw_benchmark)
       end
     after
       if File.exists?(filename), do: File.rm!(filename)
+      if File.exists?(@raw_benchmark), do: File.rm!(@raw_benchmark)
     end
   end
+
+  defp assert_header(filename) do
+    [header, _row_1, _row_2] = filename
+                             |> File.stream!
+                             |> CSV.decode
+                             |> Enum.take(3)
```

Comment:
why do we take(3) if we discard the last 2? I think just getting the first one should be enough :)

Reply:
we could assert though that there's at least one column with a value in it or so

Reply:
that's right.